lib/modules/prepareThemeSources.js

```diff
@@ -38,8 +38,9 @@ const prepareThemeSources = async function ({ rootPath }) {
       themeFiles.filter(Boolean).forEach(async (file) => {
         const sourceFilePath = path.join(PATHS.root, file).split(path.sep).filter((v, i, a) => a.indexOf(v) === i).join(path.sep);
         const destinationFilePath = sourceFilePath.slice(sourceFilePath.indexOf(TEMP_THEME_FOLDER_NAME));
+        const destinationFolderPath = destinationFilePath.split(path.sep).slice(0, -1).join(path.sep);
 
-        zipTheme.addLocalFile(sourceFilePath, destinationFilePath);
+        zipTheme.addLocalFile(sourceFilePath, destinationFolderPath);
```

Comment:
Remove file name so we pass only path to the file - thus we avoid having file name as a folder in the zip structure.